src/index.css

```diff
@@ -1,6 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Exo:wght@200&family=Inconsolata:wght@500&family=Staatliches&display=swap&family=Karma:wght@500&display=swap');
 body {
   margin: 0;
+  background-color: #0369a1;
```

Comment:
Knowing tailwind css, usually index.css is as empty as possible. Font import is ok, but for body, I'd see if you can do apply this background color directly on body in index.html